app/uk/gov/hmrc/exports/services/mapping/governmentagencygoodsitem/GovernmentAgencyGoodsItemBuilder.scala

```diff
@@ -63,19 +63,19 @@ class GovernmentAgencyGoodsItemBuilder @Inject()(
     exportItem: ExportItem,
     wcoGovernmentAgencyGoodsItem: WCOGovernmentAgencyGoodsItem,
     declarationType: DeclarationType
-  ) = {
+  ): Unit = {
     val combinedCommodity = for {
-      commodityWithoutGoodsMeasure <- mapItemTypeToCommodity(exportItem)
+      commodityWithoutGoodsMeasure <- mapExportItemToCommodity(exportItem)
       commodityOnlyGoodsMeasure = mapCommodityMeasureToCommodity(exportItem.commodityMeasure, declarationType)
     } yield combineCommodities(commodityWithoutGoodsMeasure, commodityOnlyGoodsMeasure)
 
     combinedCommodity.foreach(commodityBuilder.buildThenAdd(_, wcoGovernmentAgencyGoodsItem))
   }
-  private def mapItemTypeToCommodity(exportItem: ExportItem): Option[Commodity] =
-    (exportItem.itemType, exportItem.commodityDetails) match {
-      case (Some(item), Some(details)) =>
-        Some(cachingMappingHelper.commodityFromItemTypes(item, details, exportItem.dangerousGoodsCode, exportItem.cusCode, exportItem.taricCodes))
-      case _ => None
+  private def mapExportItemToCommodity(exportItem: ExportItem): Option[Commodity] =
+    if (exportItem.commodityDetails.isDefined) {
+      Some(cachingMappingHelper.commodityFromExportItem(exportItem))
+    } else {
+      None
     }
 
```

Comment:
This method used to be:
```
  private def mapItemTypeToCommodity(itemType: Option[ItemType]): Option[Commodity] =
    itemType.map({ item =>
      cachingMappingHelper.commodityFromItemTypes(item)
    })
```

As `ItemType` is/was a required page, the old method will always return Some(Commodity)

The new version does the same thing with `CommodityDetails` page - which is also a page that is always shown - so the effect is the same.